app/core/src/main/java/com/thechance/core/data/datasource/ProductDataSourceImp.kt

```diff
@@ -39,6 +45,18 @@ class ProductDataSourceImp : ProductDataSource, KoinComponent {
         )
     }
 
+    override suspend fun getMostRecentProducts(): List<Product> = dbQuery {
+        ProductTable
+            .select { ProductTable.isDeleted eq false }
+            .orderBy(ProductTable.id, order = SortOrder.DESC)
+            .toList()
+            .takeLast(20)
```

Comment:
Since this number is a magic number, try adding it to a constant and using that constant as the name.